apps/explorer/src/app/components/deterministic-order-details/deterministic-order-details.tsx

```diff
@@ -0,0 +1,157 @@
+import { t } from '@vegaprotocol/react-helpers';
+import { useExplorerDeterministicOrderQuery } from './__generated__/Order';
+import type { Schema } from '@vegaprotocol/types';
+import { MarketLink } from '../links';
+import PriceInMarket from '../price-in-market/price-in-market';
+import { Time } from '../time';
+
+export interface DeterministicOrderDetailsProps {
+  id: string;
+  // Version to fetch, with 0 being 'latest' and 1 being 'first'. Defaults to 0
+  version?: number;
+}
+
+const statusText: Record<Schema.OrderStatus, string> = {
+  STATUS_ACTIVE: 'Active',
+  STATUS_CANCELLED: 'Cancelled',
+  STATUS_EXPIRED: 'Expired',
+  STATUS_FILLED: 'Filled',
+  STATUS_PARKED: 'Parked',
+  // Intentionally vague - table shows partial fills
+  STATUS_PARTIALLY_FILLED: 'Active',
+  STATUS_REJECTED: 'Rejected',
+  STATUS_STOPPED: 'Stopped',
+};
+
+const sideText: Record<Schema.Side, string> = {
+  SIDE_BUY: 'Buy',
+  SIDE_SELL: 'Sell',
+};
+
+const tifShort: Record<Schema.OrderTimeInForce, string> = {
+  TIME_IN_FORCE_FOK: 'FOK',
+  TIME_IN_FORCE_GFA: 'GFA',
+  TIME_IN_FORCE_GFN: 'GFN',
+  TIME_IN_FORCE_GTC: 'GTC',
+  TIME_IN_FORCE_GTT: 'GTT',
+  TIME_IN_FORCE_IOC: 'IOC',
+};
+
+const tifFull: Record<Schema.OrderTimeInForce, string> = {
+  TIME_IN_FORCE_FOK: 'Fill or Kill',
```

Comment:
comment (non-blocking i think?): i think we ideally want these values wrapped into the dummy translate function

Reply:
Good point!